tests/fixtures/src/Request.php

```diff
@@ -176,7 +176,7 @@ public function getRoot() {
      */
     public function setRoot($root) {
         $root = trim($root, '/');
-        $root = $root ? '' : "/$root";
+        $root = $root ? "/$root" : '';
```

Comment:
This change looks very suspicious to me. Visually new version is correct.
But how this code has been working for a long time?

Reply:
Because it's a test fixture, and not the real request.

Reply:
:-)))))) sorry, missed the path of the file

Reply:
It does point to a glaring lack of coverage around around code that uses  the root for anything significant though.